java/kotlin-lite/pom.xml

```diff
@@ -0,0 +1,383 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.google.protobuf</groupId>
+    <artifactId>protobuf-parent</artifactId>
+    <version>3.15.6</version>
+  </parent>
+
+  <artifactId>protobuf-kotlin-lite</artifactId>
+  <packaging>bundle</packaging>
+
+  <name>Protocol Buffers [Lite]</name>
```

Comment:
nit: Do we want Kotlin in the name somewhere?

Reply:
I'm not sure how these naming conventions are supposed to work but I copied this from Java because it wasn't java specific either.